### PR TITLE
Create View

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/tocco-util/src/form/formBuilder.js [hunk 1]
```diff
@@ -42,6 +43,11 @@ export default (
   }
 
   const shouldRenderField = (formDefinitionField, entityField) => {
+    if (mode && formDefinitionField.scopes && formDefinitionField.scopes.length > 0
```
Comment (ottrob) [inline packages/tocco-util/src/form/formBuilder.js]: I think the check `formDefinitionField.scopes.length > 0` can be removed.

File: packages/tocco-util/src/form/reduxForm.js [hunk 2]
```diff
@@ -64,11 +72,20 @@ export const entityToFormValues = entity => {
   return result
 }
 
+export const emptyValues = modelName => {
```
Comment (ottrob) [inline packages/tocco-util/src/form/reduxForm.js]: I would prefer 

```
export const emptyValue = modelName => ({
  [wholeEntityField]: {
    entity: {
       model: modelName
    }
  }
})
```

The structure of the result object is immediately clear this way. But it's also ok if we leave it like this.

File: packages/tocco-util/src/form/reduxForm.spec.js [hunk 3]
```diff
@@ -25,9 +25,17 @@ describe('tocco-util', () => {
                 }
               }
             }
+          }
 
+          const entityModel = {
+            firstname: { },
+            gender: {relationName: 'relGender', multi: false},
+            status: {relationName: 'relGender', multi: true}
```
Comment (ottrob) [inline packages/tocco-util/src/form/reduxForm.spec.js]: replace `relGender` with `relStatus`

File: packages/entity-list/README.md [hunk 4]
```diff
@@ -21,6 +21,7 @@ React-registry name: `entity-list`
 
 ### Events
 
-| Name          | Payload                       | Description
-|---------------|-------------------------------|-------------
-| `onRowClick`  | `id` (The id of the record)   | This event is fired when a list row is clicked
+| Name                | Payload                       | Description
+|---------------------|-------------------------------|-------------
+| `onRowClick`        | `id` (The id of the record)   | This event is fired when a list row is clicked
+| `navigateToCreate`  | -                             | This event is fired when "new" button is clicked
```
Comment (ottrob) [inline packages/entity-list/README.md]: typo: ... is fired when `the` new button ...

File: packages/entity-detail/README.md [hunk 5]
```diff
@@ -11,14 +11,16 @@ React-registry name: `entity-detail`
 | `entityName`           | x         | Entity name of the record
 | `entityId`             | x         | The ID of the entity 
 | `formName`             |           | Detail-form that should be loaded
+| `mode`                 |           | Determines if the detail is in create (`create`) or update (`update`) mode
```
Comment (ottrob) [inline packages/entity-detail/README.md]: i would simplify the the description to "Determines if the detail is in create or update mode"

File: packages/entity-browser/src/routes/detail/modules/sagas.js [hunk 6]
```diff
@@ -22,18 +23,24 @@ export function* loadEntityDetail({payload}) {
   const {modelPaths, entityId, parentUrl} = yield call(parseUrl, payload.url)
   const {entityName, formBase} = yield select(entityBrowserSelector)
 
+  const mode = yield call(detail.getMode, entityId)
+
   let targetEntityName = entityName
-  let formName = `${formBase}_detail`
+  let formName = `${formBase}`
```
Comment (ottrob) [inline packages/entity-browser/src/routes/detail/modules/sagas.js]: `let formName = formBase` ?

File: packages/tocco-util/src/form/formBuilder.js (hunk 1 above)
Comment (MethodenMann) [reply packages/tocco-util/src/form/formBuilder.js]: why? by default scopes as returned by REST is an empty array. If no scope set (array empty) we dont want to regard the current mode (create or update) and display the field anyways.

File: packages/entity-browser/src/routes/detail/modules/sagas.js (hunk 6 above)
Comment (MethodenMann) [reply packages/entity-browser/src/routes/detail/modules/sagas.js]: whats is the question? :)

**Pull request merged**